tests/evaluations/task_manager/test_correctness.py

```
@@ -0,0 +1,299 @@
+    helpful_indicators = ["need", "require", "provide", "must", "should"]
+    found_indicators = [ind for ind in helpful_indicators if ind in response.lower()]
+    assert len(found_indicators) > 0, (
+        f"Response should contain helpful language: {response}"
+    )
+    
+    print(f"✓ Quality test passed - Response length: {len(response)}, "
+          f"Helpful indicators: {found_indicators}")
+
+
+@pytest.mark.asyncio
+async def test_task_manager_langsmith(pytestconfig):
```

Pylint (3.3.7)

[warning] Unused argument 'pytestconfig'

(W0613)